.github/workflows/update-deps.yml

```diff
@@ -12,19 +12,19 @@ jobs:
           node-version: lts/*
           cache: npm
       - uses: fregante/setup-git-user@v1
-
       - name: run dependency-updating script
         shell: 'script --return --quiet --command "bash {0}"'
         run: |
           npm ci
-          npm run deps:update
+          for i in {1..10}; do npm run deps:update && break || sleep 15; done
       - name: submit pull request
         uses: peter-evans/create-pull-request@v3
+        if: always()
         with:
           base: master
           branch: deps-update
           commit_message: "build(deps): bump versions"
           token: ${{ secrets.ADMIN_TOKEN }}
           title: "build(deps): bump versions"
           body: This PR was automatically generated by the update-deps GitHub action
-          labels: "chore"
+          labels: "dependencies"
```

Comment:
WDYT about keeping chore, so it matches the commit type?

Reply:
The commit type is `build` though. And I changed it so it matches the Dependabot PRs which have the `dependencies` label. I can switch it back if you want

Reply:
Gotcha. Sounds good.